L/libjulia/common.jl

```diff
@@ -17,11 +17,25 @@ function libjulia_platforms(julia_version)
         filter!(p -> !(Sys.islinux(p) && arch(p) == "powerpc64le"), platforms)
     end
 
+    for p in platforms
+        p["julia_version"] = string(julia_version)
+    end
+
+    # While the "official" Julia kernel ABI itself does not involve any C++
+    # symbols on the linker level, `libjulia` still exports "unofficial" symbols
+    # dependent on the C++ strings ABI (coming from LLVM related code). This
+    # doesn't matter if the client code is pure C, but as soon as there are
+    # other (actual) C++ dependencies, we must make sure to use the matching C++
+    # strings ABI. Hence we must use `expand_cxxstring_abis` below.
+    platforms = expand_cxxstring_abis(platforms)
+
     return platforms
 end
 
+libjulia_platforms() = [libjulia_platforms(v"1.6.0"); libjulia_platforms(v"1.7.0")]
```

Comment:
should this perhaps be the concatenation of the two vectors?

Reply:
This is actually alternative syntax for `vcat`, but I guess `vcat ` is indeed clearer.